compose-builder/Makefile

```diff
@@ -28,6 +28,7 @@ $(eval $(ARGS):;@:)
 # Order of processing these arguments is important so the overrides in the 'add' compose files occur properly.
 ifeq (dev, $(filter dev,$(ARGS)))
 	CORE_EDGEX_REPOSITORY=edgexfoundry
+	APP_SVC_REPOSITORY=edgexfoundry
```

Comment:
This needs to be is's own option. Something like `app-dev` so there is flexibility to to do either or both `dev` or `app-dev`

Reply:
changed.